framework/wazuh/manager.py

```diff
@@ -281,6 +283,8 @@ def read_ossec_conf(section: str = None, field: str = None, raw: bool = False) -
         Filters by field in section (i.e. included).
     raw : bool
         Whether to return the file content in raw or JSON format.
+    distinct : bool
+        Look for distinct values.
```

Comment:
I think this docstring is wrong. There is no distinct parameter in this function.